modules/reactor/test/tests/meshgenerators/pin_mesh_generator/pin_only.i

```diff
@@ -4,9 +4,6 @@
     dim = 2
     geom = "Square"
     assembly_pitch = 7.10315
-    axial_mesh_intervals = '1'
-    top_boundary_id = 201
-    bottom_boundary_id = 202
   []
```

Comment:
these inputs are becoming less and less usable standalone as time goes by .. we should think of a solution

Reply:
I've gone ahead and split the test input files to separate files and also checked that these are now standalone. Let me know if this works.

Reply:
it works. This will help with examples too. thank you!